migrations/versions/bfcf4182dcb5_.py

```diff
@@ -0,0 +1,43 @@
+"""empty message
```

Comment:
Will be good to add a one line on what this migration is about

Reply:
done!